tests/roles/backend_services/tasks/main.yaml

```diff
@@ -1,3 +1,46 @@
+- name: check if the source SC exists and can be used for the local-storage SC
+  no_log: "{{ use_no_log }}"
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc get sc {{ storage_class_name }} -o yaml
+  register: local_sc_result
+  failed_when:
+    - local_sc_result.rc != 0
+    - ('Error from server (NotFound)' not in local_sc_result.stdout)
+  changed_when: local_sc_result.rc != 0
```

Comment:
i had to think about that a bit... like does it need ignore_errors. but I think it is ok: 

It will fail if there is no such SC and the error is something other than "not found" eg there was some real issue talking to the kluster or some internal error there